### PR TITLE
fix(default-theme): added shipping cost to order summary

### PR DESCRIPTION
**Pull request by elkmod**
## Changes

Shipping costs weren't correctly displayed in order summary.

### CODE REVIEW

File: packages/composables/src/hooks/useCart/index.ts [hunk 1]
```diff
@@ -153,6 +154,11 @@ export const useCart = (rootContext: ApplicationVueContext): IUseCart => {
     return cartPrice || 0;
   });
 
+  const shippingTotal = computed(() => {
+    const shippingTotal = cart?.value?.deliveries[0]?.shippingCosts?.totalPrice;
```
Comment (elkmod) [inline packages/composables/src/hooks/useCart/index.ts:158]: Is optional chaining allowed?

Comment (mkucmus) [reply packages/composables/src/hooks/useCart/index.ts:158]: definitely you can, in composables the optional chaining is supported. in this case you can even use ....deliveries?.[0]? for arrays :+1: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#array_item_access_with_optional_chaining

**Review by mkucmus — APPROVED** [review]
great :1st_place_medal:

File: packages/composables/src/hooks/useCart/index.ts (hunk 1 above)
Comment (elkmod) [reply packages/composables/src/hooks/useCart/index.ts]: Alright, cool!

**Pull request merged**